### ISSUE TITLE
Could you please increase the maximum outer ring range value a little bit after joystick calibration?

### ISSUE THREAD

**Issue by lumikumi** [carpikes/ds4-tools#44]
Because the factory calibration values for the joystick are also slightly larger. If not adjusted, this can cause issues in third-person games like Hell Divers 2 or Astro Bot, where you might experience the character suddenly facing to one side during 360° movement, making the overall movement process less smooth.
Of course, if the outward extension portion can be adjusted separately, that would be even better.
Thanks again to the author and other sponsors.

**Comment by apassiou** [comment]
I want to add to this request. This is needed for when calibrating with boards as well (Jung mod), you have to overshoot by 2 or 3 clicks outside of 1.0 and -1.0 value.

**Comment by lucky0511** [comment]
yes, would be nice to have some maximum adjust, so I dont have to push the joy all the way to the edge to get max value

**Comment by Thygoo** [comment]
I would also like this option to manually adjust the range of the joysticks. After calibration I've been having stutter issues with games I didn't have this issue with before.

**Comment by qwertychouskie** [comment]
Sadly this isn't possible to implement without modifying the firmware on the controller.  https://github.com/dualshock-tools/ds4-tools/issues/23#issuecomment-1863248294
